magento/pwa-studio
# [refactor] Updated GraphQL use of UrlResolver to Route

## Description

Updates the field Query.urlResolver, which is deprecated, to use the route query instead.

## Related Issue







Closes PWA-2324

## Acceptance





Any developer

### Verification Stakeholders



Any developer

### Specification




## Verification Steps

1. Run dev server: `yarn watch:venia`
2. Verify application runs with no logging errors
3. Click through home page, category page, and product page
4. Verify no logging errors
5. Use store switcher
6. Verify store switching still works as expected






#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
